returnn/tf/util/data.py

```diff
@@ -2571,6 +2571,12 @@ def get_axes_from_description(self, axes, allow_int=True):
         return self.get_axes_from_description(list(axes))
       elif axes.startswith("stag:"):  # spatial tag
         axes = self.get_axis_by_tag_name(axes[len("stag:"):], spatial_only=True)
+      elif axes.startswith("stag-single:"):  # spatial tag which possibly matches multiple spatial axes
+        # in this case, a name of form "stag-single:<idx>:<name> is expected.
+        # idx is relative to the matching stags, i.e., it is the index among the list of spatial dims matching the name
+        idx_s, name = axes.split(":", 1)
```

Comment:
I just notice, this is wrong.
```suggestion
        _, idx_s, name = axes.split(":", 2)
```